<launch>    
  <arg name="stop_time_delay" default="1.0" />
  <arg name="check_frequency" default="5.0" />
    
  <node pkg="kuka_estop" type="kuka_estop_node" name="kuka_estop_node" output="screen">
    <param name="stop_time_delay" type="double" value="$(arg stop_time_delay)" />
    <param name="check_frequency" type="double" value="$(arg check_frequency)" />
  </node>
</launch>
